app/javascript/widget/components/ArticleListItem.vue

```diff
@@ -1,20 +1,25 @@
 <template>
   <li
-    class="py-1 flex items-center justify-between  -mx-1 px-1 hover:bg-slate-25"
+    class="py-1 flex items-center justify-between -mx-1 px-1 hover:bg-slate-75 dark:hover:bg-slate-600 rounded cursor-pointer text-slate-700 dark:text-slate-50 dark:hover:text-slate-25 hover:text-slate-900 "
+    @click="onClick"
   >
     <button
-      class="text-slate-700 hover:text-slate-900 underline-offset-2 text-sm leading-6"
+      class="underline-offset-2 text-sm leading-6 text-left"
       @click="onClick"
     >
       {{ title }}
     </button>
-    <span class="pl-1 text-slate-700 arrow">
+    <span class="pl-1 first-letter arrow">
```

Comment:
What would this do? first-letter seems to be a modifier. Without any additional classes it might not do anything

Reply:
This looks like a mistake. I have removed it.